iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/stream/project/ProjectDetailConverter.java

```diff
@@ -63,8 +62,17 @@ private static String normalizeId(String sourceId) {
             throw new RuntimeException("unexpected projectId format: " + sourceId + 
                     ", unable to split into two by " + InfoSpaceConstants.ID_NAMESPACE_SEPARATOR);
         }
-        return AbstractDNetXsltFunctions.oafId("project", 
-                tokenizedProjectId[0], tokenizedProjectId[1]); 
+        return projectOafId(tokenizedProjectId[0], tokenizedProjectId[1]);
+    }
+
+    private static String projectOafId(String prefix, String id) {
+        if (id.isEmpty() || prefix.isEmpty()) {
+            return "";
+        }
+        return projectOafSimpleId(prefix + "::" + DigestUtils.md5Hex(id));
+    }
+
+    private static String projectOafSimpleId(String id) {
+        return (InfoSpaceConstants.ROW_PREFIX_PROJECT + id).replaceAll("\\s|\\n", "");
```

Comment:
Again, this is kind of similar to `ProjectDetail` case: I am afraid we could start producing invalid identifiers once the logic gets changed on DNet side. This could be pretty difficult to spot and debug.

I would prefer to "borrow" this logic for a short period of time and rely on a common codebase once it gets published e.g. in some new lightweight dependency (like `dhp-schema` for `ProjectDetail`). So we should probably ask Claudio if it would be possible to expose this method in a different package than `dnet-openaireplus-mapping-utils` which is pretty heavy.